plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -146,6 +152,21 @@ export const createPublishGitlabMergeRequestAction = (options: {
         [tokenType]: token,
       });
 
+      const assignee = ctx.input.assignee;
+
+      let assigneeId = undefined;
+
+      if (assignee !== undefined) {
+        try {
+          const assigneeUser = await api.Users.username(assignee);
+          assigneeId = assigneeUser[0].id;
+        } catch (e) {
+          throw new InputError(
```

Comment:
Do we actually want to fail the flow here? Or rather just log that we couldn't assign it to a user but still create the MR?

Reply:
Good point, I'll make the change now